Sources/NIOHTTP1Server/main.swift

```diff
@@ -386,7 +386,7 @@ default:
     bindTarget = BindTo.ip(host: defaultHost, port: defaultPort)
 }
 
-let group = MultiThreadedEventLoopGroup(numThreads: 1)
+let group = MultiThreadedEventLoopGroup(numThreads: System.coreCount)
```

Comment:
@weissi @Lukasa I wonder if we should make this a default argument... WDYT ?

Reply:
hmm, I _feel_ (not not strongly) the user should know how many threads the want to spawn but maybe defaulting to `System.coreCount` is just fine... At the very least we should recommend this value in the docs :)

Reply:
@weissi after thinking more about it I think we should just leave it as it is (I mean no default value) as it really depends on the use-case, env and hw.